swig/include/python/ogr_python.i

```diff
@@ -103,20 +103,41 @@
             return self.GetLayerByIndex(iLayer)
         else:
             raise TypeError("Input %s is not of String or Int type" % type(iLayer))
+  }
+
+%feature("shadow") DeleteLayer %{
+    def DeleteLayer(self, value) -> "OGRErr":
+        """
+        DeleteLayer(DataSource self, value) -> OGRErr
+
+        Delete the indicated layer from the datasource.
+
+        For more details: :c:func:`OGR_DS_DeleteLayer`
+
+        Parameters
+        -----------
+            value: str | int
```

Comment:
Looks like it needs to be dedented here.

Reply:
thanks